rhobs/alerting/data_plane/prometheus.latency_release_creation_alerts.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-latency-release-creation-alerting-rules
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: latency-release-creation
+    interval: 1m
+    rules:
+    - alert: LatencyReleaseCreation
+      expr: |
+       (
+        (
+         increase(release_latency_seconds_bucket{le="+Inf"}[5m])
+         - ignoring(le)
+         increase(release_latency_seconds_bucket{le="10"}[5m])
+        ) / ignoring(le)
+        increase(release_latency_seconds_bucket{le="+Inf"}[5m])
+       ) > 0.10
+      for: 1m
+      labels:
+        severity: warning
+        slo: true
+      annotations:
```

Comment:
I suggest adding the actual query value in the summary/description. {{ $value | humanizePercentage }}